__tests__/app.test.js

```diff
@@ -61,6 +61,76 @@ describe("GET: /api/reviews/:review_id", () => {
   });
 });
 
+describe("PATCH /api/reviews/:review_id", () => {
+  test("200, responds with the updated review", () => {
+    const reviewUpdate = { inc_votes: 1 };
+    return request(app)
+      .patch("/api/reviews/1")
+      .send(reviewUpdate)
+      .expect(200)
+      .then(({ body }) => {
+        expect(body).toEqual({
+          updatedReview: {
```

Comment:
should be returned on a key of just 'review' instead of updatedReview

Reply:
Edited to review